File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
void print_row(int bricks);
int main(void)
{
    // prompt the user for the pyrmid's hight
    int n;
    do
    {
        n = get_int("Hight: ");
    }
    while(n < 1);
     // print the pyrmid of the hight
     for (int i = 0; i < n; i++)
     {
        // print row of bricks
        print_row(i + 1);
     }
}
void print_row(int bricks)
{
    // print row of bricks
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


while (True):
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break
n = 1
for i in range(height):
    for j in range(height - n):
        print(" ", end="")
    for k in range(n):
        print("#", end="")
    print("  ", end="")
    for l in range(n):
        print("#", end="")
    print("")
    n += 1
